src/about.html

```diff
@@ -21,14 +21,14 @@ <h2 class="section-heading">What We Do</h2>
 <section>
 	<div class="content text-section">
 		<h2 class="section-heading">Our Founding Team</h2>
-		<div class="col-wrap founding-team">
-			<div class="col-2">
+		<div class="row">
+			<div class="col-xs-12 col-sm-6">
 				<a href="https://uk.linkedin.com/in/vinitarathi" class="volunteer-link">
 					<img class="volunteer-picture" src="img/volunteers/Vinita_Rathi.png" alt="Vinita Rathi Picture">
 					<h2>Vinita Rathi</h2>
 				</a>
 			</div>
-			<div class="col-2">
+			<div class="col-xs-12 col-sm-6">
```

Comment:
If we have different sizes being introduced, can we have a screenshot for each, of the whole page rather than a snippet?